gui/CMakeLists.txt

```diff
@@ -165,18 +165,12 @@ target_link_libraries(TeleSculptor
   kwiver::kwiver_algo_core
   kwiver::vital_vpm
   vtkGUISupportQt
-  vtkInteractionStyle
+  vtkIOImage
   vtkIOPLY
   vtkIOXML
-  vtkViewsCore
```

Comment:
@RustyBlue I did not have an easy way of knowing which libraries were still needed, so I removed as many as I could such that it still successfully linked.  I assume several of these removed libraries are still being used and are linked as a result of dependencies of the VTK libraries I am still linking too.  Do you see any issue with removing these?  Should I leave some in to be more explicit?  Which ones?